content/boilerplates/notes/1.1.9.md

```diff
@@ -0,0 +1,10 @@
+## Bug fixes
+
+- Do not send unbounded string values to Prometheus ([Issue 14642](https://github.com/istio/istio/issues/14642)).
```

Comment:
I think it would be good to capture user impact here and one way is to use the new template that @rcaballeromx proposed.   Something like -  Eliminated the large outbound string values sent to Prometheus?